### ISSUE TITLE
Mixed content error

### ISSUE THREAD

**Issue by syed-saad-zahidi** [FraunhoferIOSB/FROST-Server#1965]
I am facing mix content error. My react application is running on https. while my frost server is on http. when creating things i am facing this error Mixed Content: The page at 'https://sensorhub-dev.hef.tum.de/devices/store#iss=https%3A%2F%2Fkeycloak-dev.hef.tum.de%2Frealms%2Fmaster' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://tuzehez-dev.srv.mwn.de:6062/FROST-Server/v1.0/Things'. This request has been blocked; the content must be served over HTTPS. I cannot put my frost server on https because there will be multiple frost instances.

**Comment by hylkevds** [comment]
Yes, mixed content is impossible in modern browsers. No way around that.
But you can run multiple FROST instances on different paths:

https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/v1.1
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-MODELV2/v1.1
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-MODELV2CORE/v1.1
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-WaterQuality/v1.1
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-GeoTech/v1.1/

All run on the same URL.

**Comment by syed-saad-zahidi** [comment]
I know it can be run on same url. But if i used this approach then there is an error from keycloak as i am using keycloak for auth.keycloak is removing /port from the redirect url and returning acess_token like url/?access_token=123. where as it should actually return url/port?access_token=123. So therefore i am trying to put my frost still on http

**Comment by hylkevds** [comment]
That sounds like a mis-configured KeyCloak instance.

Regardless, mixed content is not possible in modern browsers. There is no way around that.

**Comment by syed-saad-zahidi** [comment]
did you used keycloak to authorize these frost  instances 
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/v1.1
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-MODELV2/v1.1
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-MODELV2CORE/v1.1
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-WaterQuality/v1.1
https://ogc-demo.k8s.ilt-dmz.iosb.fraunhofer.de/FROST-GeoTech/v1.1/

if you did. Did you faced similar issue ?

**Comment by hylkevds** [comment]
Those are not secured, but in difference instances we've not had redirect issues that were not directly related to mis-configured redirect URLs in KeyCloak.
